<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>741</width>
      <height>551</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Dados</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>711</width>
        <height>481</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Placa:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="entradaPlaca"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Velocidade:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="entradaVelocidade"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Limite de velocidade:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="cb_limite">
           <item>
            <property name="text">
             <string>Selecione uma opção</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>40</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>60</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>80</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>100</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>120</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btnInserir">
         <property name="text">
          <string>Inserir</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tabelaDados">
         <column>
          <property name="text">
           <string>Placa</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Velocidade</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Situação</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Infração</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Multa</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Pontos</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>CNH</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btnOrdenarPlaca">
           <property name="text">
            <string>Ordenar por Placa</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnOrdenarVelocidade">
           <property name="text">
            <string>Ordenar por Velocidade</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btnLimparTabela">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>500</y>
        <width>221</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Limpar Tabela</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Estatísticas</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>130</y>
        <width>301</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Quantidade de veículos multados</string>
      </property>
      <widget class="QLabel" name="total_veiculos">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>161</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>250</y>
        <width>301</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Valor total das multas (R$)</string>
      </property>
      <widget class="QLabel" name="total_multas">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>22</y>
         <width>161</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>50</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>E-multas Statistics</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>739</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionSalvar"/>
    <addaction name="actionCarregar"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
  <action name="actionCarregar">
   <property name="text">
    <string>Carregar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
